client/me/purchases/manage-purchase/index.jsx

```diff
@@ -156,10 +156,45 @@ const ManagePurchase = React.createClass( {
 		);
 	},
 
+	getExpiringText( purchase ) {
+		if ( purchase.expiryStatus === 'manualRenew' ) {
+			return this.translate( '%(purchaseName)s will expire and be removed from your site in %(expiry)s. ' +
```

Comment:
This line should be changed to:

```
return this.translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s. ' +
```

to avoid a duplicate `in` (see  [screenshot](https://cldup.com/VmzS_CEZRE.png)):



Also, we should probably check this on other languages.